src/Home/index.tsx

```diff
@@ -1,59 +0,0 @@
-import React from "react"
-import { StyleSheet, ImageBackground, View } from "react-native"
-import { SvgXml } from "react-native-svg"
-import { useTranslation } from "react-i18next"
-
-import { usePermissionsContext } from "../PermissionsContext"
-import { useStatusBarEffect } from "../navigation"
-import Home from "./Home"
-
-import { Icons, Images } from "../assets"
-import { Spacing, Layout } from "../styles"
-
-const HomeScreen = (): JSX.Element => {
-  useStatusBarEffect("light-content")
```

Comment:
I am personally of two minds on the `ConnectorComponent` vs `PresentationalComponent` setup for screens. So happy with with change in general, but could you provide some context on why we are inlining the components for the home screen here?

Reply:
I'm not sure if I'm answering your question, but it didn't seem like the `Home/index` structure was adding anything useful. Seems more straightforward to just include it all in one component.

Reply:
makes sense to me!  

---

for context the reason they were separated initially was for FunctionalCore/ImperativeShell separation of concerns.  (https://www.destroyallsoftware.com/screencasts/catalog/functional-core-imperative-shell, if you are interested) Basically trying to put all the IO in spot and the rest in another as this makes it easier to test/reason about.  I agree that its a bit awkward to do this at the component level, it really wants to be at a module level, but with hooks n things that can be hard to do.

Reply:
Interesting. Thanks for this context. I haven't read much about this concept at all so will check it out tonight!